updater/dart_bindings/lib/src/bindings.dart

```diff
@@ -49,7 +58,14 @@ class AppParameters extends ffi.Struct {
     calloc.free(config.ref.base_version);
     calloc.free(config.ref.channel);
     calloc.free(config.ref.update_url);
-    calloc.free(config.ref.original_libapp_path);
+    // Free all paths in original_libapp_path.
+    var i = 0;
+    for (var i = 0; i < config.ref.original_libapp_paths_size; i++) {}
```

Comment:
?

Reply:
Oops must've accidentally changed this, will fix 👍